tests/store/db/test_utils.py

```diff
@@ -63,44 +62,43 @@ def test_alembic_escape_logic():
 
 
 def test_create_sqlalchemy_engine_with_retry_success():
-    with mock.patch.dict(os.environ, {}):
-        with mock.patch("sqlalchemy.inspect") as mock_sqlalchemy_inspect:
-            with mock.patch(
-                "mlflow.store.db.utils.create_sqlalchemy_engine"
-            ) as mock_create_sqlalchemy_engine:
-                with mock.patch("time.sleep") as mock_sleep:
-                    mock_create_sqlalchemy_engine.return_value = "Engine"
-                    engine = utils.create_sqlalchemy_engine_with_retry("mydb://host:port/")
-                    mock_create_sqlalchemy_engine.assert_called_once_with("mydb://host:port/")
-                    mock_sqlalchemy_inspect.assert_called_once()
-                    mock_sleep.assert_not_called()
-                    assert engine == "Engine"
+    with (
+        mock.patch("sqlalchemy.inspect") as mock_sqlalchemy_inspect,
+        mock.patch(
+            "mlflow.store.db.utils.create_sqlalchemy_engine"
+        ) as mock_create_sqlalchemy_engine,
+        mock.patch("time.sleep") as mock_sleep,
+    ):
```

Comment:
```suggestion
    with mock.patch("sqlalchemy.inspect") as mock_sqlalchemy_inspect,
        mock.patch("mlflow.store.db.utils.create_sqlalchemy_engine") as mock_create_sqlalchemy_engine,
        mock.patch("time.sleep") as mock_sleep:
```

I think this is the correct syntax :)

Reply:
yea I fixed the syntax :)